api/indexer.oas2.json

```diff
@@ -1126,10 +1339,6 @@
           "description": "Asset ID of the holding.",
           "type": "integer"
         },
-        "creator": {
-          "description": "Address that created this asset. This is the address where the parameters for this asset can be found, and also the address where unwanted asset units can be sent in the worst case.",
-          "type": "string"
-        },
```

Comment:
Will this impact sdk backwards compatibility? Please comment.

Reply:
Yes, similarly, if anyone was expecting this value using the SDK it will no longer compile. However, this value was always being returned as an empty string and was never implemented. So anyone who has ever written code that retrieved this field should have already found it empty and useless, and likely was not depending on it.